packages/core/src/App/Constants/routes-config.js

```diff
@@ -228,6 +230,17 @@ const initRoutesConfig = () => [
             },
         ],
     },
+    ...(isMobile()
+        ? [
+              {
+                  path: routes.complaints_policy,
+                  component: ComplaintsPolicy,
```

Comment:
should we lazy load this route too?

Reply:
For Desktop we have it as modal and for mobile its a route. Most of the content(`ComplaintsPolicyContent`) needed will be bundled with the modal. `ComplaintsPolicy` is just a wrapper component for the route with `PageOverlay`. 
If lazy load is needed, can we lazy load modal in `ToggleComplaintsPolicy` too?

Reply:
Sure I think we can do that, this Modal / route probably won't be opened all that often so no need to add the extra weight to the main bundle.